docker-compose.yaml

```diff
@@ -34,3 +34,20 @@ services:
       - 8080:80
     volumes:
       - .:/project
+
+  sawtooth-docs-redoc:
+    container_name: sawtooth-docs-redoc
+    image: sawtooth-docs-redoc
+    build:
+      dockerfile: docker/sawtooth-docs-redoc
+      context: .
+    ports:
+      - 4001:4001
+    volumes:
+      - .:/project
+    entrypoint: |
+      bash -c "
+        cd /project/docs/1.2/
+        curl https://raw.githubusercontent.com/hyperledger/sawtooth-core/1-2/rest_api/openapi.yaml -o openapi.yaml
```

Comment:
We should copy the openapi.yaml file into this repo instead of pulling it in from sawtooth-core, that's what we do in grid and splinter.

Reply:
Fixed